packages/adblocker/src/data-view.ts

```diff
@@ -389,23 +391,17 @@ export class StaticDataView {
   }
 
   public pushUTF8(raw: string): void {
-    const str = encode(raw);
-    this.pushLength(str.length);
-
-    for (let i = 0; i < str.length; i += 1) {
-      this.buffer[this.pos++] = str.charCodeAt(i);
-    }
+    const { written } = TEXT_ENCODER.encodeInto(raw, this.buffer.subarray(this.pos + 4));
+    this.pushUint32(written);
```

Comment:
I think we can change this to 16 bits (= ~65535 chars). 👍

Reply:
Reduced in [`d6032eb` (#4513)](https://github.com/ghostery/adblocker/pull/4513/commits/d6032eb11f5d52fa105f9aae2283becf75de994a)

Reply:
![image](https://github.com/user-attachments/assets/d8eb0a94-1cf2-40ea-8abd-7190432eadd5)

Can you check the min/max size of UTF8 strings at the moment? I'm wondering if this still relevant. I assume it's harder to achieve with the TextEncoder approach since we don't know in advance how many bytes will be used.

Reply:
Preferring to use `getLength` and `pushLength` as `pushUint16` is breaking the test.

Reply:
> I think they're barely called which is not my expect. I only got 5 logs:
> 
> ```
> 17
> 65
> 989
> 117
> 46
> 93
> ```

@seia-soto Did you figure out why they are barely called and if it's expected? Because if this method is almost not called maybe it's preferable to keep a simpler code for the variable length part.